docs/configuration/system/login.rst

```diff
@@ -295,6 +295,7 @@ TACACS+
 In addition to :abbr:`RADIUS (Remote Authentication Dial-In User Service)`,
 :abbr:`TACACS (Terminal Access Controller Access Control System)` can also be
 found in large deployments.
+VyOS supports only `Authentication` via the `TACACS` server. 
```

Comment:
I'd say "VyOS only supports authentication via TACACS+ servers but does not support authorization or accounting yet" to clarify the point.

Reply:
OK got it
I will change

Reply:
I made changes